data/publiccloud/log_instance.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+
+COMMAND=$1
+PROVIDER=$2
+INSTANCE_ID=$3
```

Comment:
I didn't get , we executing this within instance right ? isn't instance know it's own ID ? Maybe more correct would be get it from some variable / file on VM ?

Reply:
no it is executed from helper-vm.

Reply:
my understanding is that this script is run form our helper VM, not from the instance on the provider. Right?